src/controllers/user.controllers.js

```diff
@@ -27,6 +27,21 @@ export const createUser = async (data, response) => {
   }
 };
 
+export const userEdit = async (request, response) => {
```

Comment:
The only params passed to this function should be ones required to userEdit operation, then the results should be returned and response handled in routes.